packages/cbor/package.json

```diff
@@ -23,7 +23,7 @@
     "release": "npm version patch && git push --follow-tags && npm publish",
     "predev": "npm run coverage",
     "dev": "light-server -q -s. -w 'lib/*.js,test/*.js # npm run coverage' -o /coverage/lcov-report/index.html",
-    "types": "tsc"
+    "types": "tsc && cp -r ./types ../cbor-web/"
```

Comment:
This should work on Windows also.

Reply:
thanks for the update @hildjj 👍🏻
about your question, well I understand your point, however bundling, pollyfilling for browser is really prone to stupid errors, at least in my experience, so personally if there is a package for browser that is already available that takes care of those things, I'd rather use that, it's just easier imo.